qa/pull-tester/rpc-tests.py

```diff
@@ -255,14 +255,18 @@ def __init__(self, num_tests_parallel, test_list=None, flags=None):
         self.test_list = test_list
         self.flags = flags
         self.num_running = 0
+        # In case there is a graveyard of zombie bitcoinds, we can apply a
```

Comment:
Should we dashify `bitcoinds` here?

Reply:
We normally don't Dashify source comments. Only user faced strings and documentation is Dashified.